### PR TITLE
fix: add better support for teams and msedge

### PR DESCRIPTION
**Pull request by tardyp**
new user of komorebi, I found those two apps that are not well configured. Here is a fix for those.

Fix: https://github.com/LGUG2Z/komorebi/issues/652

- [x] I have formatted `applications.yaml` with `komorebic fmt-asc`.

and this fix the order for a previous PR. sorry about that

### CODE REVIEW

File: applications.yaml [hunk 1]
```diff
@@ -642,6 +642,14 @@
...
     comment: Dialog Box
     matching_strategy: Legacy
+- name: Microsoft Edge
+  identifier:
+    kind: Exe
+    id: msedge.exe
+    matching_strategy: Equals
+  options:
+  - tray_and_multi_window
+  - object_name_change
```
Comment (sitiom) [inline applications.yaml:652]: I don't think `object_name_change` is necessary here

File: applications.yaml [hunk 2]
```diff
@@ -702,9 +710,14 @@
...
     id: TeamsWebView
-    matching_strategy: Legacy
+    matching_strategy: Equals
   options:
   - tray_and_multi_window
+  - object_name_change
+  float_identifiers:
+  - kind: Class
+    id: TeamsWebView
+    matching_strategy: Equals
```
Comment (sitiom) [inline applications.yaml:720]: All MS Teams windows will float if this is applied,

File: applications.yaml [hunk 3]
```diff
@@ -702,9 +710,14 @@
   identifier:
     kind: Class
     id: TeamsWebView
-    matching_strategy: Legacy
+    matching_strategy: Equals
   options:
   - tray_and_multi_window
+  - object_name_change
```
Comment (sitiom) [inline applications.yaml:716]: I don't think `object_name_change` is necessary here

**Review by sitiom — CHANGES_REQUESTED** [review]

File: applications.yaml (hunk 1 above)
Comment (tardyp) [reply applications.yaml:652]: yes, edge and teams are not always sending the ShowObject event, and then komorebi fails to see the window. Franckly I am not sure the object_name_change trick is so good.
I think we should just call ShowObject if we see a visible window that is yet unknown, but this is beyond the scope of this simple PR, and I don't have all your experience on these trickeries..

File: applications.yaml (hunk 2 above)
Comment (tardyp) [reply applications.yaml:720]: I didn't find anyway to actually match for only teams call windows. I agree, after one day of test that this is annoying to just put everthing floating. I'll remove this hunk

File: applications.yaml (hunk 3 above)
Comment (PierceLovesee) [reply applications.yaml:718]: I am having the same issue as specified in [bug#652](https://github.com/LGUG2Z/komorebi/issues/652) and adding ```object_name_change``` solved it for me.  I agree with @tardyp , I think this comes down to a difference in systems.  I am running an older version of Windows 10 Enterprise (Version 10.0.19044, to be exact).

If adding ```object_name_change``` does not have any negative effects for other users, I think this change should be pushed through to main.

**Pull request closed without merging**